allennlp_models/mc/dataset_readers/fake.py

```diff
@@ -0,0 +1,89 @@
+import logging
+from typing import List
+
+from allennlp.data import DatasetReader, Instance
+from overrides import overrides
+
+logger = logging.getLogger(__name__)
+
+
+@DatasetReader.register("fake")
+class FakeReader(DatasetReader):
+    """
+    Creates fake multiple-choice input. If your model doesn't get 99% on this data, it is broken.
+
+    Instances have two fields:
+     * ``alternatives``, a ListField of TextField
```

Comment:
Super super annoying nit-pick, but all of these double backticks should just be single backticks or they won't render properly in the docs when we address https://github.com/allenai/allennlp/issues/4124 😬